### PR TITLE
Makin quickstart section more look like terminal

### PR DESCRIPTION
**Pull request by christoga**
Hello Mr Juan,
In this pull request I'm making your 'Quickstart` section more look like terminal/cmd

It will be awesome if you accept this pull request, thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ Legofy is a python program that takes a static image or gif and makes it so that
...
 ### Quickstart
-* `git clone https://github.com/JuanPotato/Legofy.git`
-* `cd Legofy`
-* `python setup.py install`
-* `legofy image.jpg` - anywhere
+```
+$ git clone https://github.com/JuanPotato/Legofy.git
+$ cd Legofy
+$ python setup.py install
+$ legofy image.jpg - anywhere
```
Comment (nabijaczleweli) [inline README.md]: Maybe `s/-/#/` on this line, since it's a shell script?

**Comment by christoga** [comment]
I think its more better to use $ 😁 like https://github.com/id-python/direktori

**Comment by JuanPotato** [comment]
Yeah it looks nice

**Comment by christoga** [comment]
Thanks mr @JuanPotato , Hoping you remember me in telegram 😇

**Comment by JuanPotato** [comment]
I immediately recognized your profile picture :smile:

**Comment by nabijaczleweli** [comment]
Wergh, I was suggesting using `#` instead of `-` on that line, because it's a comment in a shell script

**Comment by JuanPotato** [comment]
@nabijaczlewli oooh,

**Pull request merged**